.github/workflows/check-url.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+if [ $# -ne 1 ]; then
+    exit 1;
+fi
+
+filename=$(cut -d ";" -f 1 <<< "$1")
+URL=$(cut -d ";" -f 2 <<< "$1")
+
+status_code=$(curl --write-out '%{http_code}' --silent  --output /dev/null "$URL")
```

Comment:
I don't see it documented in `man curl`, but I think it supports `--` for marking the end of options (so, the beginning of the url), which is theoretically safer.
```suggestion
status_code=$(curl --write-out '%{http_code}' --silent  --output /dev/null -- "$URL")
```